2022-02-17 12:25:55,275 INFO    MainThread:12380 [internal.py:wandb_internal():87] W&B internal server running at pid: 12380, started at: 2022-02-17 12:25:55.275161
2022-02-17 12:25:55,279 DEBUG   HandlerThread:12380 [handler.py:handle_request():130] handle_request: check_version
2022-02-17 12:25:55,279 INFO    WriterThread:12380 [datastore.py:open_for_write():77] open: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/run-re6owfsk.wandb
2022-02-17 12:25:55,284 DEBUG   SenderThread:12380 [sender.py:send():238] send: header
2022-02-17 12:25:55,284 DEBUG   SenderThread:12380 [sender.py:send_request():252] send_request: check_version
2022-02-17 12:25:55,366 DEBUG   SenderThread:12380 [sender.py:send():238] send: run
2022-02-17 12:25:55,631 INFO    SenderThread:12380 [dir_watcher.py:__init__():169] watching files in: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files
2022-02-17 12:25:55,631 INFO    SenderThread:12380 [sender.py:_start_run_threads():825] run started: re6owfsk with start time 1645100754
2022-02-17 12:25:55,631 DEBUG   SenderThread:12380 [sender.py:send():238] send: summary
2022-02-17 12:25:55,632 INFO    SenderThread:12380 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2022-02-17 12:25:55,634 DEBUG   HandlerThread:12380 [handler.py:handle_request():130] handle_request: run_start
2022-02-17 12:25:55,646 DEBUG   HandlerThread:12380 [meta.py:__init__():40] meta init
2022-02-17 12:25:55,646 DEBUG   HandlerThread:12380 [meta.py:__init__():54] meta init done
2022-02-17 12:25:55,646 DEBUG   HandlerThread:12380 [meta.py:probe():214] probe
2022-02-17 12:25:55,655 DEBUG   HandlerThread:12380 [git.py:repo():33] git repository is invalid
2022-02-17 12:25:55,655 DEBUG   HandlerThread:12380 [meta.py:_save_pip():58] save pip
2022-02-17 12:25:55,656 DEBUG   HandlerThread:12380 [meta.py:_save_pip():72] save pip done
2022-02-17 12:25:55,656 DEBUG   HandlerThread:12380 [meta.py:_save_conda():79] save conda
2022-02-17 12:25:55,664 ERROR   HandlerThread:12380 [meta.py:_save_conda():88] Error saving conda packages
Traceback (most recent call last):
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/site-packages/wandb/sdk/internal/meta.py", line 84, in _save_conda
    subprocess.call(
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/subprocess.py", line 340, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/sohui/anaconda3/envs/unetr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda'
2022-02-17 12:25:55,667 DEBUG   HandlerThread:12380 [meta.py:_save_conda():89] save conda done
2022-02-17 12:25:55,667 DEBUG   HandlerThread:12380 [meta.py:probe():252] probe done
2022-02-17 12:25:55,850 DEBUG   SenderThread:12380 [sender.py:send():238] send: files
2022-02-17 12:25:55,851 INFO    SenderThread:12380 [sender.py:_save_file():960] saving file wandb-metadata.json with policy now
2022-02-17 12:25:55,862 DEBUG   HandlerThread:12380 [handler.py:handle_request():130] handle_request: stop_status
2022-02-17 12:25:55,863 DEBUG   SenderThread:12380 [sender.py:send_request():252] send_request: stop_status
2022-02-17 12:25:56,494 INFO    Thread-11 :12380 [upload_job.py:push():137] Uploaded file /tmp/tmpvhex9m24wandb/6fdr67u6-wandb-metadata.json
2022-02-17 12:25:56,635 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_created():217] file/dir created: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/wandb-summary.json
2022-02-17 12:25:56,635 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_created():217] file/dir created: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/output.log
2022-02-17 12:25:56,636 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_created():217] file/dir created: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/wandb-metadata.json
2022-02-17 12:25:56,636 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_created():217] file/dir created: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/requirements.txt
2022-02-17 12:25:56,637 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_created():217] file/dir created: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/conda-environment.yaml
2022-02-17 12:25:56,667 WARNING MainThread:12380 [internal.py:wandb_internal():146] Internal process interrupt: 1
2022-02-17 12:25:58,632 INFO    Thread-8  :12380 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/output.log
2022-02-17 12:25:58,739 WARNING MainThread:12380 [internal.py:wandb_internal():146] Internal process interrupt: 2
2022-02-17 12:25:58,739 ERROR   MainThread:12380 [internal.py:wandb_internal():149] Internal process interrupted.
2022-02-17 12:25:58,882 INFO    SenderThread:12380 [sender.py:finish():1091] shutting down sender
2022-02-17 12:25:58,883 INFO    WriterThread:12380 [datastore.py:close():281] close: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/run-re6owfsk.wandb
2022-02-17 12:25:58,883 INFO    HandlerThread:12380 [handler.py:finish():731] shutting down handler
2022-02-17 12:25:58,883 INFO    SenderThread:12380 [dir_watcher.py:finish():283] shutting down directory watcher
2022-02-17 12:25:59,634 INFO    SenderThread:12380 [dir_watcher.py:finish():313] scan: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files
2022-02-17 12:25:59,634 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/output.log output.log
2022-02-17 12:25:59,635 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/wandb-metadata.json wandb-metadata.json
2022-02-17 12:25:59,635 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/config.yaml config.yaml
2022-02-17 12:25:59,636 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/conda-environment.yaml conda-environment.yaml
2022-02-17 12:25:59,645 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/requirements.txt requirements.txt
2022-02-17 12:25:59,645 INFO    SenderThread:12380 [dir_watcher.py:finish():327] scan save: /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/wandb-summary.json wandb-summary.json
2022-02-17 12:25:59,655 INFO    SenderThread:12380 [file_pusher.py:finish():177] shutting down file pusher
2022-02-17 12:25:59,656 INFO    SenderThread:12380 [file_pusher.py:join():182] waiting for file pusher
2022-02-17 12:26:00,235 INFO    Thread-13 :12380 [upload_job.py:push():137] Uploaded file /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/config.yaml
2022-02-17 12:26:00,282 INFO    Thread-14 :12380 [upload_job.py:push():137] Uploaded file /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/requirements.txt
2022-02-17 12:26:00,294 INFO    Thread-15 :12380 [upload_job.py:push():137] Uploaded file /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/wandb-summary.json
2022-02-17 12:26:00,308 INFO    Thread-12 :12380 [upload_job.py:push():137] Uploaded file /home/sohui/code/DTC/code/wandb/run-20220217_122554-re6owfsk/files/output.log
2022-02-17 12:26:00,851 INFO    MainThread:12380 [internal.py:handle_exit():77] Internal process exited
